### DISCUSSION TITLE
Cannot get podman 4.0 on Fedora 35

### DISCUSSION THREAD

**Discussion by djnotes** [containers/podman#13430] [General]
I enabled updates-testing repo for fedora following the official doc, but I cannot get Podman 4.0. The latest version available is still 3.4.4. 
Are new versions of Podman available in updates-testing? By the way, I am running Fedora 35 on WSL 2.

**Comment by baude** [comment]
We cannot release 4.0 on fedora 35 because it has breaking changes.  you can use the copr repo -> https://copr.fedorainfracloud.org/coprs/rhcontainerbot/podman-next/

**Reply by Luap99** [reply]
use https://copr.fedorainfracloud.org/coprs/rhcontainerbot/podman4/ for podman v4, podman-next has the latest main builds which can be less stable and are not as well tested

**Comment by rhatdan** [comment]
We should write a blog on this and add a note to getting Podman on podman.io on this.

**Comment by djnotes** [comment]
Thanks. Is this also breaking? 
 
 
  On Sun, Mar 6, 2022 at 17:42, Paul ***@***.***> wrote:   


use https://copr.fedorainfracloud.org/coprs/rhcontainerbot/podman4/ for podman v4, podman-next has the latest main builds which can be less stable and are not as well tested

**Comment by baude** [comment]
https://github.com/containers/podman.io/pull/505
